### PR TITLE
Apply server.ps1 fixes to all PS/CLI container images

### PR DESCRIPTION
**Pull request by lamw**
Closes: #549
Signed-off-by: William Lam <wlam@vmware.com>

## Pull Request Checklist
🚨 Please review the [guidelines for contributing](https://vmweventbroker.io/community) to this repository.

- [ ] Please ensure that you are making a pull request against the **Development** branch
- [ ] Please use the `WIP` keyword in the title of your PR if you are not ready for review
- [ ] Please ensure that you have opened a Github Issue if you are resolving/fixing a problem
- [ ] Please ensure that you have [signed](https://help.github.com/en/github/authenticating-to-github/signing-commits) all commits and that you have [squashed](https://medium.com/@slamflipstrom/a-beginners-guide-to-squashing-commits-with-git-rebase-8185cf6e62ec) all relevant commits related to your change
- [ ] Please make sure that you have tested your change locally by successfully [building and deploying the VMware Event Broker Appliance](https://vmweventbroker.io/kb/contribute-appliance) and/or [building and deploying VMware Event Router](https://vmweventbroker.io/kb/contribute-eventrouter)
- [ ] Please include any relevant screenshots and/or output as part of your testing
- [ ] Please include any documentation updates that is applicable for your changes

## Change Type

What types of changes does your code introduce to the VMware Event Broker Appliance?

_Put an `x` in all boxes that apply_

Please check the type of change your PR introduces:
- [x] Bugfix (non-breaking change which fixes an issue)
- [ ] Feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Code style update (formatting, renaming)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] Documentation changes
- [ ] Other (please describe):

## Testing Verification

* Verified PS fixes
* Successfully built and pushed all PS/PCLI images to VMware Harbor registry

## Additional Information

* Any other details you wish to include or mention

If you have any questions/comments, feel free to reach out to team on Slack [#vcenter-event-broker-appliance](https://vmwarecode.slack.com/archives/CQLT9B5AA)

Thank you from the VEBA Team! 🥳

### CODE REVIEW

**Comment by embano1** [comment]
Is 

`FROM photon@sha256:ec5b5e7bf027963c146ad22e411c18d288fa1bacf847f592603864d00e14c95c` the most recent?

File: examples/knative/powershell/kn-ps-echo/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,4 @@
-FROM projects.registry.vmware.com/veba/ce-ps-base:1.1
-ENV TERM linux
-ENV PORT 8080
+FROM projects.registry.vmware.com/veba/ce-ps-base:1.2🤯
```
Comment (embano1) [inline examples/knative/powershell/kn-ps-echo/Dockerfile:1]: Are you serious with that emoji :D ?

File: examples/knative/powercli/kn-pcli-tag/README.md [hunk 2]
```diff
@@ -149,4 +149,4 @@ kubectl -n vmware-functions delete -f function.yaml
 
 # delete secret
 kubectl -n vmware-functions delete secret tag-secret
-```
\ No newline at end of file
+```
```
Comment (embano1) [inline examples/knative/powercli/kn-pcli-tag/README.md:152]: Should we also bump the docker image tags in this file?

**Comment by embano1** [comment]
All fixes in server.ps1 itself are already in, right?

File: examples/knative/powershell/kn-ps-echo/Dockerfile (hunk 1 above)
Comment (lamw) [reply examples/knative/powershell/kn-ps-echo/Dockerfile:1]: unintentional copy/paste 😂, will fix

**Comment by lamw** [comment]
> All fixes in server.ps1 itself are already in, right?

Yup!

File: examples/knative/powercli/kn-pcli-tag/README.md [hunk 3]
```diff
@@ -6,7 +6,8 @@ Example Knative PowerCLI function for applying a vSphere Tag when a Virtual Mach
 Create the container image locally to test your function logic.
 
 ```
-docker build -t <docker-username>/kn-pcli-tag:1.0 .
+export TAG=<version>
```
Comment (embano1) [inline examples/knative/powercli/kn-pcli-tag/README.md:9]: At least in this file, there's still a 1.0 reference.

**Review by embano1 — CHANGES_REQUESTED** [review]
Still 1.0 refs in READMEs

**Comment by lamw** [comment]
> Still 1.0 refs in READMEs

@embano1 sorry, I must have forgot to amend changes. Should be good now!

**Review by embano1 — APPROVED** [review]
LGTM

**Comment by embano1** [comment]
TODO:

- [ ] Build and push all template images and examples

**Comment by lamw** [comment]
> Is
> 
> `FROM photon@sha256:ec5b5e7bf027963c146ad22e411c18d288fa1bacf847f592603864d00e14c95c` the most recent?

No, the latest PH 3.x is `photon:3.0-20210821` - https://hub.docker.com/layers/photon/library/photon/3.0-20210821/images/sha256-cb58bd1e7b6652da68995ca4fc14b7832684e4e38c8f194776287c98c24e8c91?context=explore

Do we also want to bump this?

**Pull request merged**